### PR TITLE
Task #10801: Administrators add categories

### PR DESCRIPTION
**Pull request by huynhlv**
Screenshot from 2019-04-19 17-15-58
LinkRedmi: https://edu-redmine.sun-asterisk.vn/issues/10801

### CODE REVIEW

**Comment by huynhlv** [comment]
ready

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Admin::CategoriesController < ApplicationController
+  before_action :mod_or_admin
+  before_action :load_category, except: %i(create new)
+  def new
```
Comment (VuThiTranVan) [inline app/controllers/admin/categories_controller.rb]: Thêm line trống trước line này

File: app/views/admin/categories/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<%= form_for [:admin, @category] do |f| %>
+	<%= render "shared/error_messages", object: f.object %>
+
+	<%= f.label :name, t(".name") %>
+	<%= f.text_field :name, class: "form-control" %>
+
+	<%= f.submit yield(:button_text), class: "btn btn-primary" %>
```
Comment (VuThiTranVan) [inline app/views/admin/categories/_form.html.erb]: Indent..

**Comment by huynhlv** [comment]
ready

**Pull request merged**